### ISSUE TITLE
Ability to continuous processing of stream from camera

### ISSUE THREAD

**Issue by navrkald** [Microsoft/FFmpegInterop#122]
Hello,
I would need to capture camera stream then pipeline it to Ffmpeg and then pipeline it to program for next processing.
Only what I am able to do is save data from camera to stream, stop recording and then pass stream to ffmpeg.
If I pass stream without stop recording I get error from ffmpeg...
Any advice?
Thanks.

**Comment by khouzam** [comment]
Yes, this should be possible, It seems to be a similar issue as #131.

If you have a link that you could share it would be easier for us to try it.
